plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -13,18 +13,18 @@ import (
 	"sync"
 	"time"
 
-	dialout "github.com/cisco-ie/nx-telemetry-proto/mdt_dialout"
-	telemetry "github.com/cisco-ie/nx-telemetry-proto/telemetry_bis"
-	"github.com/golang/protobuf/proto" //nolint:staticcheck // Cannot switch to "google.golang.org/protobuf/proto", "github.com/golang/protobuf/proto" is used by "github.com/cisco-ie/nx-telemetry-proto/telemetry_bis"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	_ "google.golang.org/grpc/encoding/gzip" // Register GRPC gzip decoder to support compressed telemetry
 	"google.golang.org/grpc/peer"
+	"google.golang.org/protobuf/proto"
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/metric"
 	internaltls "github.com/influxdata/telegraf/plugins/common/tls"
 	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/cisco_telemetry_mdt/dialout"
+	"github.com/influxdata/telegraf/plugins/inputs/cisco_telemetry_mdt/telemetry"
```

Comment:
Why do you need to move these to telegraf. I would feel better if we are using the upstream versions instead of copying those into telegraf adding a maintenance burden...

Reply:
I will see if it is possible to generate the `.pb.go` files from vendored `.proto` files instead of copied ones, though these `.pb.go` files will have to be held onto since the repos we are referencing either do not have generated forms of the files, or provide generated forms using outdated libraries